cellarium/ml/models/incremental_pca.py

```diff
@@ -26,8 +30,8 @@ class IncrementalPCA(CellariumModel, PredictMixin):
        <https://www.cs.toronto.edu/~dross/ivt/RossLimLinYang_ijcv.pdf>`_.
 
     Args:
-        g_genes:
-            Number of genes.
+        feature_schema:
```

Comment:
Please rename `feature_schema` here and elsewere to `feature_names`. The word "schema" is reserved for a greater scope of data structuring and I am anticipating that we will run into name collisions in the future.